types/sharp/index.d.ts

```diff
@@ -927,7 +927,7 @@ declare namespace sharp {
         /** Use high quality chroma subsampling (optional, default false) */
         smartSubsample?: boolean | undefined;
         /** Level of CPU effort to reduce file size, integer 0-6 (optional, default 4) */
-        reductionEffort?: number | undefined;
```

Comment:
I agree with @rexxars, these should be marked with `@deprecated` rather than being removed outright.